deluge/common.py

```diff
@@ -720,6 +720,23 @@ def parse_human_size(size):
     raise InvalidSize(msg % (size, tokens))
 
 
+def wrap_url_with_html(text):
+    """
+    A simple regex sub to wrap all occurrences of URLs with HTML
+
+    Args:
+        text (str): The string to replace urls in.
```

Comment:
With Python 3 type hints can be moved to function definition

Reply:
Sure, I'm just used to JSDocs in typescript and saw it used in other functions in common, so I just copy pasted it and did the small editing you see. 

Will move to type hints in the definition though.